overlord/hookstate/ctlcmd/fde_setup.go

```diff
@@ -60,9 +60,9 @@ func init() {
 }
 
 func (c *fdeSetupRequestCommand) Execute(args []string) error {
-	context := c.context()
-	if context == nil {
-		return fmt.Errorf("cannot run fde-setup-request without a context")
```

Comment:
I think including snapctl subcommand name in the message was nice/useful; maybe "'snapctl %s' can only be invoked from within a snap"?

Reply:
Done!